pkg/uwb-core/dpl/config.c

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
I think I tried to do this and could not, but I'll look at it again to give a proper answer.

Reply:
Ok so the reason why I didn't is because the needed function is defined `https://github.com/Decawave/uwb-core/blob/master/porting/dpl_os/src/config.c`, but this file defines a lot of functions I don't really need.

What I could do is add a `porting/riot` directory in the package and hide these files as a patch that I can include from there. Would that be better? I can do the same for the `qeue.h` header

Reply:
> hide these files as a patch that I can include from there

This I don't get.

Reply:
Do be able to include them from the package I would need to create new files ot new directories so I can include them withouth including other headers that would mess up everything.

Reply:
See 2ce77cd185816fcc8e5529992b22d87df08cc38d